roles/cfg_openwrt/templates/corerouter/config/airos-dfs-reset.j2

```diff
@@ -1,8 +1,15 @@
 #jinja2: trim_blocks: "true", lstrip_blocks: "true"
+{% if airos_dfs_reset|default([])|length > 0 %}
+config airos-dfs-reset general
+	option interval		'600'
+	option reset_sleep	'900'
+{% endif %}
```

Comment:
bitte auch den loop mit einschließen.

konsequenterweise kannst du dann auch `|default([])` in zeile 8 entfernen

sorry for nitpicking